cockroachcloud/cloud-org-sso.md

```diff
@@ -0,0 +1,126 @@
+
+- For members who log in in using an IdP managed by your organization, deprovisioning them from the IdP prevents them from accessing  resources in your {{ site.data.products.db }} organization.
+- For members who log in using an identity you don't manage, such as a personal GMail account, you must deprovision them from {{ site.data.products.db }} to prevent their access to your {{ site.data.products.db }} organization.
+
+### Autoprovisioning
+
+Autoprovisioning allows you to centralize management of your users in an IdP and removes the need to [invite users to your organization](https://www.cockroachlabs.com/docs/cockroachcloud/console-access-management.html#invite-team-members-to-cockroachdb-cloud). When autoprovisioning is enabled, the first time a new user successfully signs in using the custom sign-in page, a {{ site.data.products.db }} account is automatically created for them and and are assigned the [Developer role](/docs/cockroachcloud/console-access-management.html#developer) by default.
+
+When autoprovisioning is enabled for an SSO authentication method, a {{ site.data.products.db }} account is created automatically when a member successfully authenticates using that method. Together with [Allowed Email Domains](configure-cloud-org-sso.html#allowed-email-domains), autoprovisioning allows the users who are already provisioned in your IdP to get started quickly, without waiting for an invitation.
+
+Autoprovisioning is optional, and can be configured separately for each enabled SSO authentication method.
+
+{{site.data.alerts.callout_danger}}
+{{ site.data.products.db }} users are identified by their email address. To reduce the risk of duplicated users, ensure that users have unique email addresses before enabling auto-provisioning for an authentication method. If duplicate users result from auto-provisioning, you must delete them manually. Refer to [Manage Team Members](/docs/cockroachcloud/console-access-management.html#manage-team-members).
+{{site.data.alerts.end}}
+
+Cockroach Labs recommends that you enable autoprovisioning on only a single SSO method at a time, and that you migrate your users gradually. Most organizations aim to manage users in a single centralized IdP. It may be necessary to temporarily enable autoprovisioning to migrate a group of users who have not yet been created in your centralized IdP.
+
+### Migration of individual members to SSO
+
+After you [enable an authentication method](configure-cloud-org-sso.html#enable-or-disable-an-authentication-method) for your organization, it will appear on your organization's custom URL, and your existing users can opt to sign in using that method rather than their password. When an existing member signs in using an SSO authentication method for the first time, they can optionally designate that authentication method as their new default.
```

Comment:
```suggestion
After you [enable an authentication method](configure-cloud-org-sso.html#enable-or-disable-an-authentication-method) for your organization, it will appear on your organization's custom URL. Your existing users can then sign in using that method rather than a method they may have been using earlier. When an existing member signs in using an SSO authentication method for the first time, they can optionally designate that authentication method as their new default.
```